umap/static/umap/js/modules/importer.js

```diff
@@ -1,147 +1,165 @@
 import { DomUtil, DomEvent } from '../../vendors/leaflet/leaflet-src.esm.js'
 import { translate } from './i18n.js'
+import Dialog from './ui/dialog.js'
+
+const AVAILABLE_IMPORTERS = ['geodatamine', 'communesfr', 'presets']
```

Comment:
I don't get why we need this instead of using `UMAP_EXPERIMENTAL_IMPORTERS`?

Reply:
Good point: I've done this as we have a dynamic import so I wanted to make sure we would only load known path. Does that make sense or I'm overthinking that thing ?

Reply:
I can't see how it can be dangerous with the current implementation.